sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -201,6 +202,16 @@ protected static final DataType getSparkType(DType type) {
     }
   }
 
+  protected static final DataType getSparkTypeFrom(ColumnViewPointerAccess access) {
+    DType type = access.getDataType();
+    if (type == DType.LIST) {
+      // TODO should have nullability in there differently
```

Comment:
Curious what you are thinking for this TODO.

Reply:
Not sure if nullability really matters in this context for the type.